python/ray/tune/tests/test_tuner_restore.py

```diff
@@ -628,7 +628,7 @@ def train_func_1(config):
     with pytest.raises(ValueError):
         tuner = Tuner.restore(
             str(tmpdir / "overwrite_trainable"),
-            trainable="__fake",
+            trainable="abcd",
```

Comment:
nit: Not a big deal but standardizing on `MOCK_TRAINABLE_NAME` might make it a bit easier to change in the future.

Reply:
this one is also a dummy value that is not actually used.